### PR TITLE
init commit

### PR DESCRIPTION
**Pull request by Ekamba**

### CODE REVIEW

File: react/week1/Todo-Liste/src/App.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+// import components
```
Comment (Daryan-Tech) [inline react/week1/Todo-Liste/src/App.js]: you don't have to write a comment for the packages that you import

File: react/week1/Todo-Liste/src/App.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+import React from "react";
+
+// Create components
```
Comment (Daryan-Tech) [inline react/week1/Todo-Liste/src/App.js]: please write a more clear message, it is more important to know what this component does.

File: react/week1/Todo-Liste/src/App.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  render() {
+    // Array of <todoList>
+    const listItems = this.state.tasks.map(task => (
+      <Task key={task.taskId} title={task.title} when={task.when} />
+    ));
+    return <ul className="task-list">{listItems}</ul>;
+  }
+}
+
+// export
```
Comment (Daryan-Tech) [inline react/week1/Todo-Liste/src/App.js]: and here you don't have to create a comment for when you export the component.

**Comment by Daryan-Tech** [comment]
good job 👍 

please just have in mind that you want to create a comment that will describe the new code that you have written, not the standard stuff that comes by default like the export and the import stuff.

File: react/week1/Todo-Liste/src/App.js (hunk 2 above)
Comment (Ekamba) [reply react/week1/Todo-Liste/src/App.js]: Thanks for your advices

**Pull request merged**